patroni/api.py

```diff
@@ -293,6 +293,16 @@ def do_GET_metrics(self):
         metrics.append("# TYPE patroni_dcs_last_seen gauge")
         metrics.append("patroni_dcs_last_seen{0} {1}".format(scope_label, postgres.get('dcs_last_seen', 0)))
 
+        metrics.append("# HELP patroni_pending_restart Value is 1 if cluster need a restart, 0 otherwise.")
+        metrics.append("# TYPE patroni_pending_restart gauge")
+        metrics.append("patroni_pending_restart{0} {1}"
+                       .format(scope_label, int(patroni.postgresql.pending_restart == True)))
+
+        metrics.append("# HELP patroni_is_paused Value is 1 if auto failover is disabled, 0 otherwise.")
+        metrics.append("# TYPE patroni_is_paused gauge")
+        metrics.append("patroni_is_paused{0} {1}"
+                       .format(scope_label, int(patroni.ha.is_paused() == True)))
```

Comment:
flake8 complains about these lines: https://github.com/zalando/patroni/runs/5048624670?check_suite_focus=true#step:5:9

Reply:
Oopsie, my bad. It's fixed.